src/akkudoktoreos/prediction/price_forecast.py

```diff
@@ -1,144 +0,0 @@
-import hashlib
```

Comment:
to reset to a specific commit, try:
```
git reset --hard 213aa7b3d1a067980b3d4600e15e79205b02d873
```
or use revert. (You can find unreferenced commits with git reflog, they are available for 30 days. To use that commit on some branch, you can git cherry-pick xyz)

Reply:
I didn´t say anything

Reply:
Sorry, this is my first PR... I try my best :-)

Reply:
I have replace the price_forcast.py, but i don't know how can i remove the price_forecast.py from the PR.

Reply:
no worries, just reset to the Akkudoktor main branch and then put your commit on top and update this branch (force push):
```
# make sure to have Akkudoktor-EOS as remote "akkudoktor" in your git repo
git remote add akkudoktor https://github.com/Akkudoktor-EOS/EOS.git

# reset your current main branch to the akkudoktor github
git reset --hard akkudoktor/main

# Get your previous commit with the state changes (can look it up with git reflog) and push
git cherry-pick 340ad9da53ad3784f368bf8b95afaa0030e9057c
git push --force
```

For the other PR best create a feature branch (aka not the main branch, so you can keep that in sync with the akkudoktor repo)
```
# get latest changes of the akkudoktor github
git fetch akkudoktor

# create a new branch "prices"
git checkout -b prices akkudoktor/main

# Get your previous commit with the price calculation
git cherry-pick b886ef3c026a982423bed2093d6f42f072893fef

# Push to new branch (which is currently active, so it is also referenced as HEAD) on your fork
git push -u origin HEAD
```